sdks/python/apache_beam/examples/ml-orchestration/kfp/components/ingestion/src/ingest.py

```diff
@@ -0,0 +1,57 @@
+"""Dummy ingestion function that fetches data from one file and simply copies it to another."""
+
+import argparse
+from pathlib import Path
+import time
+
+
+def parse_args():
+  """Parse ingestion arguments."""
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+    "--ingested-dataset-path", type=str,
+    help="Path to save the ingested dataset to.")
+  parser.add_argument(
+    "--base-artifact-path", type=str,
+    help="Base path to store pipeline artifacts.")
+  return parser.parse_args()
+
+
+def dummy_ingest_data(
```

Comment:
Can we add something like `# [START ingest_component]` and highlight this section in the `orchestration.md` docs?